README.md

```diff
@@ -74,6 +75,14 @@ sudo yum install azure-cli-2.0.69-1.el7.x86_64.rpm
 
 When running Muchos under Ubuntu 18.04, checkout these [tips](docs/azure-ubuntu-1804.md).
 
+You might also need to configure the `aws_ami` property in [muchos.props]. Muchos by default uses a free
+CentOS 7 image that is hosted in the AWS marketplace but managed by the
+CentOS organization. If you have never used this image in EC2 before, you will need to go to the
+[CentOS 7 product page][centos7] to accept the software terms. If this is not done, you will get an
+error when you try to launch your cluster. By default, the `aws_ami` property is set to an AMI in `us-east-1`.
+You will need to changes this value if a newer image has been released or if you are running in different region
+than `us-east-1`.
+
```

Comment:
I'm not sure it makes sense to move this to the Azure section. Should probably go back to where it was originally, right?

Reply:
To add to this, the default ami in mucho.props.example should probably be updated to the most recent ami. It has been at its current version for a while but it is still different than the default one given in our props file.

Reply:
> I'm not sure it makes sense to move this to the Azure section. Should probably go back to where it was originally, right?

Maybe that isn't the right spot, but it gets overlooked in the original location. Do you have a suggestion if we were to move that paragraph, where it would sound more logical? Or I could put it back to its original location.

Reply:
I second the concern around moving this note into the Azure section; that would further increase chances of overlooking it, and potentially confuse Azure users. Perhaps, just add emphasis for this step of using the correct AMI, using Markdown syntax, within the EC2 section itself?

Reply:
> To add to this, the default ami in mucho.props.example should probably be updated to the most recent ami. It has been at its current version for a while but it is still different than the default one given in our props file.

Can you create a subsequent PR after this to do that? Thanks.